src/overlays/actors/ovl_En_Fish2/z_en_fish2.c

```diff
@@ -22,95 +65,1112 @@ const ActorInit En_Fish2_InitVars = {
     (ActorFunc)EnFish2_Draw,
 };
 
-// static ColliderJntSphElementInit sJntSphElementsInit[2] = {
-static ColliderJntSphElementInit D_80B2B318[2] = {
+static ColliderJntSphElementInit sJntSphElementsInit[2] = {
     {
-        { ELEMTYPE_UNK2, { 0xF7CFFFFF, 0x00, 0x00 }, { 0xF7CFFFFF, 0x00, 0x00 }, TOUCH_NONE | TOUCH_SFX_NORMAL, BUMP_NONE, OCELEM_ON, },
+        {
+            ELEMTYPE_UNK2,
+            { 0xF7CFFFFF, 0x00, 0x00 },
+            { 0xF7CFFFFF, 0x00, 0x00 },
+            TOUCH_NONE | TOUCH_SFX_NORMAL,
+            BUMP_NONE,
+            OCELEM_ON,
+        },
         { 1, { { 0, 0, 0 }, 0 }, 1 },
     },
     {
-        { ELEMTYPE_UNK2, { 0xF7CFFFFF, 0x00, 0x00 }, { 0xF7CFFFFF, 0x00, 0x00 }, TOUCH_NONE | TOUCH_SFX_NORMAL, BUMP_NONE, OCELEM_ON, },
+        {
+            ELEMTYPE_UNK2,
+            { 0xF7CFFFFF, 0x00, 0x00 },
+            { 0xF7CFFFFF, 0x00, 0x00 },
+            TOUCH_NONE | TOUCH_SFX_NORMAL,
+            BUMP_NONE,
+            OCELEM_ON,
+        },
         { 17, { { 0, 0, 0 }, 0 }, 1 },
     },
 };
 
-// static ColliderJntSphInit sJntSphInit = {
-static ColliderJntSphInit D_80B2B360 = {
-    { COLTYPE_HARD, AT_NONE, AC_NONE, OC1_ON | OC1_TYPE_ALL, OC2_TYPE_1, COLSHAPE_JNTSPH, },
-    2, D_80B2B318, // sJntSphElementsInit,
+static ColliderJntSphInit sJntSphInit = {
+    {
+        COLTYPE_HARD,
+        AT_NONE,
+        AC_NONE,
+        OC1_ON | OC1_TYPE_ALL,
+        OC2_TYPE_1,
+        COLSHAPE_JNTSPH,
+    },
+    2,
+    sJntSphElementsInit,
 };
 
-#endif
+static f32 D_80B2B370[] = { 0.01f, 0.012f, 0.014f, 0.017f };
+static f32 D_80B2B380[] = { 0.019f, 0.033f };
+
+void func_80B28370(EnFish2* this, s32 arg0) {
+    static AnimationHeader* D_80B2B388[] = {
+        &D_060013AC, &D_060007D4, &D_060006D8, &D_060006D8, &D_06001174, &D_06000ACC,
+    };
+    static u8 D_80B2B3A0[] = { 0, 0, 2, 2, 2, 2, 0, 0 };
```

Comment:
```suggestion
    static AnimationHeader* D_80B2B388[] = {
        &D_060013AC, &D_060007D4, &D_060006D8, &D_060006D8, &D_06001174, &D_06000ACC,
    };
    static u8 D_80B2B3A0[] = { 0, 0, 2, 2, 2, 2 };
```
There are 6 animation entries, so the rest is just padding